infra/main.tf

```diff
@@ -102,3 +102,58 @@ resource "aws_cloudwatch_event_target" "cloudwatch_event_target" {
     "table" : aws_dynamodb_table.dynamodb_table.name
   })
 }
+
+resource "aws_elasticsearch_domain" "elasticsearch_domain" {
+  domain_name           = "tf-creds-es-domain"
+  elasticsearch_version = "7.10"
+
+  cluster_config {
+    instance_type = "t3.small.elasticsearch"
+  }
+
+  ebs_options {
+    ebs_enabled = true
+    volume_size = 10
+  }
+
+  node_to_node_encryption {
+    enabled = true
+  }
+
+  encrypt_at_rest {
+    enabled = true
+  }
+
+  domain_endpoint_options {
+    enforce_https       = true
+    tls_security_policy = "Policy-Min-TLS-1-2-2019-07"
+  }
+
+  advanced_security_options {
+    enabled                        = true
+    internal_user_database_enabled = true
+    master_user_options {
+      master_user_name     = "elasticsearch"
```

Comment:
same for suggestion of adding it to variable.tf also here and for the password (line 137)

Reply:
+1